src/Subsystems/Chassis.h

```diff
@@ -20,17 +24,30 @@ class Chassis: public frc::Subsystem {
 
 public:
 
+
+
 	enum ShifterValue{kShifterHigh = DoubleSolenoid::kForward, kShifterLow = DoubleSolenoid::kReverse};
+	enum AutoShifterValue{kAutoShiftOn = true, kAutoShiftOff = false};
```

Comment:
Yes it is used in `Commands/ToggleAutoShifting.cpp` however this is the only place it is used.